src/Integration.Vsix/ErrorList/IssuesSnapshot.cs

```diff
@@ -147,7 +147,7 @@ private IssuesSnapshot(Guid snapshotId, string projectName, string filePath, IEn
 
         public override bool TryGetValue(int index, string keyName, out object content)
         {
-            if (index < 0 || this.issues.Count <= index)
+            if (index < 0 || index >= issues.Count || !issues[index].Span.HasValue || issues[index].Span.Value.IsEmpty)
```

Comment:
I think the new condition deserves a comment as it's not obvious why we're doing it e.g.
`If the requested issue doesn't have a valid span we return false so the Error List won't display it`